Earthfile

```diff
@@ -93,15 +94,15 @@ hack:
   COPY --platform=${USERPLATFORM} +helm-setup/helm .hack/helm
   COPY --platform=${USERPLATFORM} +kind-setup/kind .hack/kind
   COPY (+helm-build/output --CROSSPLANE_VERSION=v0.0.0-hack) .hack/charts
```

Comment:
i was considering updating the version everywhere, but there's a bit of inconsistency in how we treat the helm vs. image vs. binary version, so I haven't touched the helm version at all because it's not causing any problems for me so far 😇